lib/config/presets/internal/regex-managers.ts

```diff
@@ -14,4 +14,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  githubActionsVersions: {
+    description:
+      'Update `_VERSION` environment variables in GitHub Action files.',
+    regexManagers: [
+      {
+        fileMatch: ['^.github/workflows/[^\\.]+\\.ya?ml$'],
+        matchStrings: [
+          '# renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>[^\\s]+?)(?: (lookupName|packageName)=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s  .+?_VERSION\\s*:\\s*["\']?(?<currentValue>.+?)["\']?\\s',
```

Comment:
double space?
```suggestion
          '# renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>[^\\s]+?)(?: (lookupName|packageName)=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s .+?_VERSION\\s*:\\s*["\']?(?<currentValue>.+?)["\']?\\s',
```

Reply:
it should be `\s+` as it depends on tab width settings

Reply:
indeed, that would be more portable. I've been using two-space indent everywhere, but for tabs users that would probably make sense.

Reply:
should be fixed by the latest commit :)